.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -39,7 +39,7 @@ body:
   attributes:
     label: Renderer
     options:
-      - DX9 (default)
-      - DX11
+      - DX11 (default)
+      - DX9
```

Comment:
```suggestion
      - DX9 (Legacy)
```
Would like to be very clear that it is legacy. Also fits the startup options
![image](https://user-images.githubusercontent.com/9014762/145690271-29d066f4-ba6d-4a1e-ae68-022cb0488ee2.png)